inference-engine/include/ie_plugin_config.hpp

```diff
@@ -92,6 +92,7 @@ DECLARE_METRIC_KEY(FULL_DEVICE_NAME, std::string);
  *
  * The possible values:
  *  - "FP32" - device can support FP32 models
+ *  - "BF16" - device can support BF16 computations for FP32 and BF16 models
```

Comment:
BTW, do we really have BF16 models? I though it's just a computational mode.

Reply:
There is a possibility to mark BF16 in IR for edges manually. That case and if KEY_ENFORCE_BF16=NO and on CPX, we can execute IR with BF16 layers